test/ctia/entity/incident_test.clj

```diff
@@ -40,6 +40,12 @@
                                :headers {"Authorization" "45c1f5e3f05d0"})]
            (is (= 200 (:status response)))))
 
+       (testing "Bad route returns 404"
+         (let [response (GET app
+                             (str "ctia/incident/" (:short-id incident-id) "/tactics")
+                             :headers {"Authorization" "45c1f5e3f05d0"})]
+           (is (= 404 (:status response)))))
```

Comment:
then it returns a 404 :-)?

Reply:
Yeah, I got nowhere investigating this.

Reply:
This probably happens because of a misconfig at the reverse proxy level (caddy)